Source/Include/swigwarn.h

```diff
@@ -315,6 +315,16 @@
 
 /* please leave 890-899 free for Go */
 
-/* -- User defined warnings (900 - 999) -- */
+#define WARN_OBJC_TYPEMAP_OBJCTYPE_UNDEF     900
+#define WARN_OBJC_TYPEMAP_IMTYPE_UNDEF       901
+#define WARN_OBJC_TYPEMAP_OBJCIN_UNDEF       902
+#define WARN_OBJC_TYPEMAP_OBJCOUT_UNDEF      903
+#define WARN_OBJC_MULTIPLE_INHERITANCE       904
+#define WARN_OBJC_INTERFACE_MOD              905
+#define WARN_OBJC_IMPLEMENTATION_MOD         906
+
+/* please leave 900-919 free for ObjectiveC */
+
+/* -- User defined warnings (1000 - 999) -- */
```

Comment:
As it says higher up "Language module specific warnings (700 - 899)", AIUI 900 to 999 is reserved for the user.

It looks like there are unused ranges within the language module range - e.g. you could use 760-779.

>